resources/osl_mysql_dev.rb

```diff
@@ -0,0 +1,31 @@
+provides :osl_mysql_dev
+resource_name :osl_mysql_dev
```

Comment:
@osuosl-cookbooks/chefs thoughts on this name instead?
```suggestion
resource_name :osl_mysql_test
```

Reply:
Change to the requested name